src/ScatterPlot.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { ScatterPlot } from '@nivo/scatterplot';
+
+const Scatterplot = ({ graphData, minAxis }) => (
+	<>
+		<div
+			style={{
+				display: 'flex',
```

Comment:
There is a lot of inline style here that doesn't seem to change dynamically. Can we assign them to CSS classes in `style.less` instead?